packages/react-app/src/components/ChallengeCard.jsx

```diff
@@ -1,16 +1,20 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { Typography } from "antd";
 
-const ChallengeCard = ({ challengeInfo, submissionInfo }) => (
+const { Text, Link: AntdLink } = Typography;
+
+const ChallengeCard = ({ challengeId, challengeInfo, submissionInfo }) => (
   <div style={{ display: "inline-block", opacity: challengeInfo.disabled ? 0.5 : 1 }}>
     {challengeInfo.disabled ? (
-      <span>{challengeInfo.label}</span>
+      <Text>{challengeInfo.label}</Text>
     ) : (
-      <a href={challengeInfo.url} target="_blank" rel="noreferrer noopener">
+      <Link to={`/challenge/${challengeId}`} component={AntdLink}>
```

Comment:
For some reaseon, having `component={AntdLink}` makes the page reload when clicking the link.

Reply:
That's true. Not sure why they recommend replacing the component then... https://ant.design/components/typography/#How-to-use-Typography.Link-in-react-router

I'll take that out.